src/Data/WideWord/Word128.hs

```diff
@@ -434,6 +456,67 @@ pokeElemOff128 ptr idx (Word128 a1 a0) = do
   pokeElemOff (castPtr ptr) (2 * idx + index0) a0
   pokeElemOff (castPtr ptr) (2 * idx + index1) a1
 
+-- -----------------------------------------------------------------------------
+-- Functions for `Prim` instance.
+
+{-# INLINE sizeOf128# #-}
+sizeOf128# :: Word128 -> Int#
+sizeOf128# _ = 2# *# sizeOf# (undefined :: Word64)
+
+{-# INLINE alignment128# #-}
+alignment128# :: Word128 -> Int#
+alignment128# _ = alignment# (undefined :: Word64)
```

Comment:
Don't think that is correct :).

Reply:
why not?

Reply:
Alignment for `Word128` should be `2 * alighnment (undefined :: Word64)` shouldn't it?

Reply:
Oh yeah. Oops